smol/moca/sublattice.py

```diff
@@ -176,6 +176,14 @@ def get_index(sp, species):
             part_sublattices.append(part_sublatt)
         return part_sublattices
 
+    def __eq__(self, other):
+        """Check equality of sublattices."""
+        if other.site_space == self.site_space and np.array_equal(
+            other.sites, self.sites
+        ):
```

Comment:
I suppose sublattice encoding needs to be checked as well?

Reply:
Thanks! added checking encoding as well